profiles/common/modules/features/nexteuropa_newsroom/classes/NewsroomAgenda.inc

```diff
@@ -0,0 +1,679 @@
+<?php
+
+/**
+ * @file
+ * Code for the NewsroomAgenda class.
+ */
+
+/**
+ * NewsroomAgenda class base class.
+ */
+abstract class NewsroomAgenda {
+
+  const AGENDA_VIEW_SCHEDULE = 'schedule';
+  const AGENDA_VIEW_MONTH = 'month';
+  const AGENDA_VIEW_WEEK = 'week';
+  const AGENDA_VIEW_DAY = 'day';
+  const AGENDA_VIEW_SCHEDULE_BLOCK = 'schedule-block';
+
+  protected $view = NewsroomAgenda::AGENDA_VIEW_SCHEDULE;
+  protected $topicUrl = 'all';
+  protected $topics = array();
+  protected $typeUrl = 'all';
+  protected $typeId = NULL;
+  protected $currentDate = NULL;
+  protected $nextDate = NULL;
+  protected $previousDate = NULL;
+  protected $currentItems = array();
+  protected $nextItems = array();
+  protected $result = array();
+  protected $query = NULL;
+  protected $itemsCount = 0;
+  protected $urlParts = array();
+  protected $displayCalendar = TRUE;
+  protected $isBlock = FALSE;
+
+  /**
```

Comment:
Can you check one more time? Or explain one more time what do you want exactly.

This is an example, how getters and setters documentation has been improved.
`
/**
   * Gets view.
   *
   * @return string
   *   Type of view.
   */
  public function getView() {
    return $this->view;
  }

  /**
   * Sets view.
   *
   * @param string $value
   *   Type of view.
   */
  public function setView($value) {
    $this->view = $value;
  }
`